src/Auditor.jl

```diff
@@ -209,10 +209,10 @@ function audit(prefix::Prefix, src_name::AbstractString = "";
         rm(f; force=true)
     end
 
-    # Make sure that `prefix` in pkg-config files use a relative prefix
+    # Make sure that `(exec_)prefix` in pkg-config files use a relative prefix
     pc_files = collect_files(prefix, endswith(".pc"))
-    pc_re = r"^prefix=(/.*)$"
-    for f in pc_files
+    for f in pc_files, prefix in ["prefix", "exec_prefix"]
```

Comment:
`prefix` should have a different name, it's shadowing another variable with the same name.